new-backend/app/modules/langgraph_nodes/store_and_send.py

```
@@ -1,12 +1,15 @@
+# from app.utils.vector_store import save_to_vector_db
+from app.modules.vector_store.chunk_rag_data import chunk_rag_data
 
 
 def store_and_send(state):
     # to store data in vector db
     try:
-        save_to_vector_db({
-            **state
-        })
+        print(state)
+        chunks = chunk_rag_data(state)
```

Ruff (0.11.9)

Local variable `chunks` is assigned to but never used

Remove assignment to unused variable `chunks`

(F841)